gc/verbose/VerboseManager.hpp

```diff
@@ -129,7 +129,11 @@ class MM_VerboseManager : public MM_VerboseManagerBase
 	 * @param[in] env the current environment.
 	 * @return void
 	 */
-	virtual void prepareForCheckpoint(MM_EnvironmentBase *env) { closeStreams(env); }
+	virtual void prepareForCheckpoint(MM_EnvironmentBase *env)
+	{
```

Comment:
To match formatting of other functions in this file.
```suggestion
	virtual void
	prepareForCheckpoint(MM_EnvironmentBase *env)
	{
```